packages/angular_devkit/architect/src/api.ts

```diff
@@ -300,16 +300,22 @@ async function handleAsyncIterator<T>(
 /**
  * A builder handler function. The function signature passed to `createBuilder()`.
  */
-export interface BuilderHandlerFn<A> {
+export interface BuilderHandlerFn<Options, InfrastructureSettings = {}> {
```

Comment:
Following yesterdays discussion, we should not invoke the builder using architect, instead we should use the same pattern like we currently use in the dev-server. Thus, I think that changes the fact that we don't need to add this a public API.

Reply:
Yep, reverted all the Architect changes in favor of using a `buildEsbuildBrowserInternal()` entry point with the extra options so we don't need to change `InfrastructureSettings`.